app/controllers/concerns/sufia/works_controller_behavior.rb

```diff
@@ -7,7 +7,6 @@ module WorksControllerBehavior
 
     included do
       self.show_presenter = Sufia::WorkShowPresenter
-      layout "sufia-one-column"
```

Comment:
Does this controller behavior not need `with_themed_layout '1_column'` or is that accounted for elsewhere?